vita3k/gui/src/controllers_dialog.cpp

```diff
@@ -282,9 +283,58 @@ void draw_controllers_dialog(GuiState &gui, EmuEnvState &emuenv) {
                             ImGui::EndTable();
                         }
                         ImGui::EndTable();
+                    }
 
-                        ImGui::End();
+                    const SDL_JoystickGUID guid = SDL_JoystickGetDeviceGUID(i);
+                    if (ctrl.controllers[guid].has_led) {
+                        const auto set_color_led = [&](const std::vector<int> led) {
+                            SDL_GameControllerSetLED(ctrl.controllers[guid].controller.get(), led[0], led[1], led[2]);
+                            config::serialize_config(emuenv.cfg, emuenv.cfg.config_path);
+                        };
+                        ImGui::Spacing();
+                        ImGui::SetCursorPosX((ImGui::GetWindowWidth() / 2.f) - (ImGui::CalcTextSize("Color Led").x / 2.f));
+                        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Color Led");
+                        auto &led = emuenv.cfg.controller_led_value;
+                        bool is_using_custom_led = !led.empty();
+                        if (ImGui::Checkbox("Using Custom Color", &is_using_custom_led)) {
+                            const std::vector<int> default_led = { 0, 0, 65 };
+                            if (led.empty())
+                                led = default_led;
+                            else
+                                led.clear();
+                            set_color_led(default_led);
```

Comment:
This doesn't make any sense

Reply:
? Is for reset default color, is logical 🤷‍♂️

Reply:
is for can delete/set default color
set custom color -> click on check box -> disable custom color with reset default color and save config